README.md

```diff
@@ -1 +1,38 @@
-# example-rust
\ No newline at end of file
+# Examples
+
+This repository houses an example Alvarium Publisher and Receiver application split into 2 packages.
+They can be run from the same machine or not, but they are 2 independent packages that encompass 
+the lifecycle of a data source annotating and being scored on the other end. 
+
+## Publishing
+The publisher package provides an example of an application that produces data from 2 mock sensors. 
+This data is generated randomly and produces an sdk instance that contains a set of core annotators 
+from the Alvarium rust [SDK](https://github.com/DyrellC/alvarium-sdk-rust). Additionally, it creates 
```

Comment:
@DyrellC Review this PR as well for old repo names. Both the annotator and SDK have been merged at this point.